examples/mobile/ScheduleApp/second-list.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, user-scalable=no">
+	<title>Scheduling App</title>
+	<link href="lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link rel="stylesheet" href="css/style.css" />
+	<script data-build-remove="false" src="lib/tau/mobile/js/tau.js"></script>
+	<link href="css/DayOfWeekPicker.css" rel="stylesheet" />
+	<script src="js/DayOfWeekPicker.js"></script>
+	<script src="js/day-indicator.js"></script>
+</head>
+
+<body>
+	<div class="ui-page ui-page-active" id="second-list">
+		<header>
+			<div class="ui-appbar-left-icons-container">
+				<a href="#main" class="ui-btn ui-btn-icon ui-btn-icon-back" data-style="flat"></a>
```

Comment:
Is it correct the "#main" link  for back key? 
Shouldn't back key go to index.html?

Reply:
In theory it shouldn't matter because `index.html` has page with id = `main` but flow in router is different. When we know that this page has already exists the page can be changed without load.

Reply:
oh ok. I see.
Thank you for your kind explanation.